### ISSUE TITLE
Docs : Missing Stdlib entries

### ISSUE THREAD

**Issue by leinadsepol** [skiplang/skip#40]
If we go on [matches function of String doc](http://www.skiplang.com/docs/string.html#matches)
Links to the regular expression module are available.
However, the documents targeted by these links are not accessible.

In addition, like Regexp, some other very useful modules like Subprocess, Math, ... are not present in the Stdlib documentation

**Comment by josephsavona** [comment]
Yeah, the stdlib docs on the website only cover the core types. There are a few features that need to be added to the docs in order to handle more types. A big one is showing the public fields of a class, since some of the classes (off the top of my head, the return value of `Subprocess.spawn`) have fields only and no methods. Help is definitely appreciated here!
